src/Asserts/Dependencies/ObjectDependenciesDescription.php

```diff
@@ -30,15 +30,19 @@ public static function make(string $path): ?self
             return null;
         }
 
-        /** @var Node\Stmt\UseUse[] $useUses */
-        $useUses = ServiceContainer::$nodeFinder->findInstanceOf($description->stmts, Node\Stmt\UseUse::class);
+        /** @var Node\Name [] $names */
+        $names = ServiceContainer::$nodeFinder->findInstanceOf($description->stmts, Node\Name::class);
 
         $description->uses = new ObjectUses(
             array_map(
-                static function (Node\Stmt\UseUse $useUse): string {
-                    return $useUse->name->toCodeString();
+                static function (Node\Name $nodeName): string {
+                    $name = $nodeName->toCodeString();
```

Comment:
Seems that stuff such as `parent`, or `true` or being captured here.

Reply:
and native function names, but I think it is dependents too

Reply:
Do you plan to work on this soon? Or should I try to address this?

Reply:
I hope this doesn't have a negative effect

Reply:
Well, at least things like `true` or `null`, can't be considered as "dependents". That's basically native PHP code.

Reply:
@ta-tikoma  You think you can address the comment above soon, or you wish that I complete this pull request myself?

Reply:
@nunomaduro sorry for long time. make your pull request, please

Reply:
@ta-tikoma If you wish - you can also add me as maintainer of this package, and we can maintainer together.

Reply:
Perfect. I will address this pull request after the 15 February. I am busy until then.